hitb-ctf-amsterdam-2016/crypto/capture-mexico-tls-200/README.md

```diff
@@ -0,0 +1,18 @@
+# HITB CTF Amsterdam 2016 : Can you break TLS?
+
+**Category:** Crypto
+**Points:** 200
+**Solves:** 1
+**File:** [tls_16970cb3b09a9dd01f5b82449d9c1795.tar.gz](https://github.com/zack-lau/WriteUps/blob/master/HITB-Facebook-CTF-2016/capture_Mexico-tls/tls_16970cb3b09a9dd01f5b82449d9c1795.tar.gz)
```

Comment:
woop, I think I just understood you, you meant the file not the other write up located on a different github repo :]